server/app/views/BaseHtmlView.java

```diff
@@ -99,6 +99,11 @@ protected String renderDateTime(Instant time, ZoneId zoneId) {
     return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd 'at' h:mm a z"));
   }
 
+  protected String renderDate(Instant time, ZoneId zoneId) {
+    ZonedDateTime dateTime = time.atZone(zoneId);
+    return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd"));
+  }
+
```

Comment:
suggestion (blocking): what do you think of moving this into `services.DateConverter`?

Reply:
Done.

Reply:
nice, thanks!